packages/cfd/build/constants.js

```diff
@@ -25,6 +25,7 @@ const {
 } = require('./loaders-config');
 
 const ALIASES = {
+    'react/jsx-runtime': 'react/jsx-runtime.js',
```

Comment:
Question: what's the purpose of this react alias? Why do we need this here?

Reply:
@shahzaib-deriv without this line I can't use `deriv-com/ui` library, I have an issue and can't compile the package.